docs/Developers/Review.rst

```diff
@@ -117,6 +117,15 @@ For the Reviewers
    behaviour well. E.g. for bear tests just testing for a good and bad file
    is **not** sufficient.
 
+.. note::
+
+    While reviewing Pull request/``Patch`` for ``difficulty/low`` issue make
+    sure the Patch solves the issue and does't create any further issues.
```

Comment:
Spell Check: "and **doesn't** create any further issues."
@RaiVaibhav please rectify the spelling error

Reply:
Changes Done